src/pages/questions/QuestionDisplay.jsx

```diff
@@ -57,13 +57,67 @@ const getValueTagExamplesURL = (question) => {
   return "";
 };
 
-const QuestionDisplay = ({ question, answerQuestion }) => {
+const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
   const { t } = useTranslation();
   const valueTagQuestionsURL = getValueTagQuestionsURL(question);
   const valueTagExamplesURL = getValueTagExamplesURL(question);
 
   if (question === NO_QUESTION_LEFT) {
-    return <p>{t("questions.no_questions_remaining")}</p>;
+    return (
+      <Stack alignItems="center" spacing={2}>
+        <Stack
+          direction={{ xs: "column", sm: "row" }}
+          alignItems="center"
+          spacing={{ xs: 0, sm: 2 }}
+        >
+          <h2>{t("questions.no_questions_remaining")}</h2>
+          <Button size="small" variant="contained" onClick={resetFilters}>
+            Reset filters
+          </Button>
+        </Stack>
+        <h1>OR</h1>
```

Comment:
`<h1>` has a meaning: this is the top title of the page. I assume the top title of this page is not "OR" 😉

Same for `h2`, `h3`, ... If you want a big text use CSS, not HTML

On this topic, you can read [`<Typography/>` doc](https://mui.com/material-ui/react-typography/#main-content)

```jsx
<Typography variant="h1" component="p">A big text</Typography>
```

This components hase two important props:
- `variant`: defines how the text will look like (here it will look like a top title)
- `component`the HML compont which will be used (the sementic meaning (here it's a basic text)

Reply:
Oh okay, got it. Thank you. I'll make the changes